shell/imports/client/signup-client.js

```diff
@@ -20,13 +20,22 @@ import { Meteor } from "meteor/meteor";
 import { Template } from "meteor/templating";
 import { Router } from "meteor/iron:router";
 import { DEFAULT_SIGNUP_DIALOG } from "/imports/client/personalization.js";
-import { globalDb } from "/imports/db-deprecated.js";
+import { SandstormDb } from "/imports/sandstorm-db/db.js";
 
 Template.signup.helpers({
   signupDialog: function () {
-    const setting = globalDb.collections.settings.findOne("signupDialog");
+    const setting = SandstormDb.collections.settings.findOne({_id: "signupDialog"});
```

Comment:
Looks like you missed this file when addressing my previous comment.

Reply:
Yep.  Just fixed it, and built.  A great example of how more eyes is good.